contrib/devtools/test-security-check.py

```
@@ -39,6 +39,17 @@ def env_flags() -> List[str]:
         flags += filter(None, os.environ.get(var, '').split(' '))
     return flags
 
+def call_security_check(cc: str, source: str, executable: str, options) -> tuple:
```

Ruff (0.12.2)

Unused function argument: `cc`

(ARG001)